src/components/composite/MarginAmount/MarginAmount.tsx

```diff
@@ -50,12 +50,16 @@ const MarginAmount: React.FunctionComponent<MarginAmountProps> = ({
     : defaultMargin;
   const initialMarginRequirement = currentPositionMarginRequirement || defaultMargin;
 
-  const maxAmountToWithdraw =
+  let maxAmountToWithdraw =
     !isUndefined(marginAmount) && !isUndefined(initialMarginRequirement)
-      ? marginAmount - initialMarginRequirement
+      ? Math.max(marginAmount - initialMarginRequirement, 0)
       : undefined;
+  if (maxAmountToWithdraw) {
+    maxAmountToWithdraw = Math.floor(maxAmountToWithdraw * 10000) / 10000;
+  }
```

Comment:
you can merge this Math.max and Math.floor in one line and make maxAmountToWithdraw a const. WDYT?